jans-cedarling/flask-sidecar/main/v1/resource.py

```diff
@@ -22,3 +22,15 @@ def post(self, payload):
                                 payload.get("context", {})
                         )
         return auth_response
+
+@blp.route("/.well-known/authzen-configuration")
+class WellKnown(MethodView):
+    @blp.response(200, WellKnownSchema)
+    def get(self):
+        """
+        Returns authzen configuration endpoint
+        """
+        response = {
+            "access_evaluation_v1_endpoint": "http://127.0.0.1:5000/cedarling/evaluation"
```

Comment:
Why it is hard-coded localhost?

Reply:
The sidecar is always supposed to be called from the same host i.e. not over the internet. Each cloud service would have its own deployment of the flask sidecar, and it would be called on localhost. I can make it dynamic so that it returns hostname instead for future proofing